rero/rero-ils-ui
# locations: implement detail view

- NEW Implements detail view for locations.
- BETTER detail view for locations.

Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>

## Why are you opening this PR?

Implements task 1180 of US 987

## How to test?

- `git checkout rero/pr/38`
- If needed: `npm install`
- `npm run start-admin-proxy` (needs localhost:5000 to be served in https and login as admin or system librarian)
- Go to libraries page and test several location detail views
- Test to delete a location (not online) and see the message displayed

## Code review check list
